apps/web/src/components/PreJoinScreens/PreJoinScreens.tsx

```diff
@@ -70,6 +79,8 @@ export default function PreJoinScreens() {
           await playerConnect(data.token);
           registerUserDocument(data.sync_object_names.user_document);
           registerSpeakersMap(data.sync_object_names.speakers_map);
+          registerViewersMap(data.sync_object_names.viewers_map);
```

Comment:
We'll need to register the raised hands map here too.

Reply:
This could even refactor this
```
        registerRaisedHandsMap(data.sync_object_names.raised_hands_map);
        registerSpeakersMap(data.sync_object_names.speakers_map);
        registerViewersMap(data.sync_object_names.viewers_map);
```
Into something like `registerSyncMaps(data.sync_object_names)` if you want.

Reply:
cool, took a stab at this. curious to see your thoughts and if you were thinking of handling it a different way